contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+community. There is also a risk that the RFC process as designed will not
+sufficiently address the scaling challenges we face today. PR review bandwidth is
+already at a premium and we may find that the RFC process introduces an unreasonable
+bottleneck on our development velocity.
+
+It certainly can be argued that the lack of a dedicated issue/defect tracker
+beyond GitHub issues contributes to our challenges in managing a project as large
+as Kubernetes, however, given that other large organizations, including GitHub
+itself, make effective use of GitHub issues perhaps the argument is overblown.
+
+The centrality of Git and GitHub within the RFC process also may place too high
+a barrier to potential contributors, however, given that both Git and GitHub are
+required to contribute code changes to Kubernetes today I would argue that we
+should focus on providing support to those unfamiliar with this tooling.
+
+Expanding the proposal template beyond the single sentence description currently
+required in the [features issue template][] may be a heavy burden for non native
+English speakers and here I believe that the role of the RFC editor combined with
+kindness and empathy will be crucial to making the process successful.
+
+[features issue template]: https://github.com/kubernetes/features/blob/master/ISSUE_TEMPLATE.md
+
+## Alternatives
+
+This RFC process is related to
+- the generation of our [architectural road map][]
+- the fact that the [what constitutes a feature][] is still undefined
+- how we [manage issues][]
+- the difference between an [accepted design and a proposal][]
+- [the organization of design proposals][]
+
+and I believe that this RFC process attempts to place these concerns within a
+more general framework
+
+[architectural road map]: https://github.com/kubernetes/community/issues/952
+[what constitutes a feature]: https://github.com/kubernetes/community/issues/531
+[manage issues]: https://github.com/kubernetes/community/issues/580
+[accepted design and a proposal]: https://github.com/kubernetes/community/issues/914
+[the organization of design proposals]: https://github.com/kubernetes/community/issues/918
+
+## Unresolved Questions
+
+Whether we believe it makes sense for a SIG repository to serve a mini mono
+repository where code lives in a structure like
+
+```
+kubernetes:sig-node/pkg/kublet
+```
+
+or whether SIGs maintain a list of repositories maintained by the SIG in an index
+like
+
+```
+kubernetes:sig-node/repositories.md
+```
+
+is not clear to me. I believe that users would generally think about Kubernetes
+from a component standpoint if they are aware of the component at all but I have
+no real data to support that assertion.
+
+## Mentors
+
+- caleb miles
+  - github: [calebamiles](https://github.com/calebamiles/)
+  - slack: [calebamiles](https://coreos.slack.com/team/caleb.miles)
+  - email: [caleb.miles@coreos.com](mailto:caleb.miles@coreos.com)
+  - pronoun: "he"
```

Comment:
Caleb, this is an excellent proposal.  Thank you for being so thorough and thoughtful, as always.